cdap-tms/src/main/java/co/cask/cdap/messaging/service/CoreMessagingService.java

```diff
@@ -247,6 +267,29 @@ protected void startUp() throws Exception {
     LOG.info("Core Messaging Service started");
   }
 
+  /**
+   * Creates the given topic if it is not yet created. Adds a topic to the creationFailureTopics if creation fails.
+   */
+  private void createSystemTopic(String topicName,  Queue<TopicId> creationFailureTopics) {
+    TopicId topicId;
+    try {
```

Comment:
This results in variable not initialized for topicId, since `topicId = NamespaceId.SYSTEM.topic(topicName)` is not guaranteed to complete in this case